receiver/netflowreceiver/factory.go

```diff
@@ -0,0 +1,64 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package netflowreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/netflowreceiver"
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/receiver"
+)
+
+var (
+	typeStr = component.MustNewType("netflow")
+)
```

Comment:
```suggestion
```
This is generated for you as the  `Type` variable inside `internal/metadata/generated_status.go`.